odk1-src/aggregate-vm.rst

```diff
@@ -1,4 +1,65 @@
-ODK Aggregate Virtual Machine 
-=================================
+.. spelling::
 
-The `ODK Aggregate VM <https://gumroad.com/l/odk-aggregate-vm>`_ is a fully-configured install of Aggregate that you can run on any computer. It requires very little setup, works well without Internet connectivity, and gives you complete control over your data collection campaign.
+  readme
+  VirtualBox
+
+ODK Aggregate Virtual Machine
+=============================
+
+This document provides instructions on setting up ODK Aggregate using the `ODK Aggregate VM OVA file <https://github.com/opendatakit/aggregate/releases/download/v1.5.0-beta.0/ODK-Aggregate-v1.5.0-beta.0.ova.zip>`_ and `VirtualBox <https://www.virtualbox.org>`_.
+
+.. admonition:: Before you get started…
+
+  - Please read this entire document before installing the virtual machine (VM). Doing so will save you a lot of time and effort.
+
+Basic Usage
+-----------
+
+1. Download and install VirtualBox from `virtualbox.org <https://www.virtualbox.org>`_.
+2. Double-click `ODK-Aggregate-v1.5.0-beta.0.ova <https://github.com/opendatakit/aggregate/releases/download/v1.5.0-beta.0/ODK-Aggregate-v1.5.0-beta.0.ova.zip>`_ to import the VM. Accept the default settings.
+3. After the import completes, start the VM.
+4. After the VM starts, wait for Welcome to ODK Aggregate VM message to be shown.
+5. Do not login into the black and white screen of the VM. Instead on your computer, open a web browser.
+6. Go to the web address shown in the VM boot screen (e.g., `http://localhost:10080 <http://localhost:10080>`_). You will see the Aggregate login screen.
+7. Click "Sign in with Aggregate password" and login with username and password shown below.
```

Comment:
"Sign in with Aggregate password" is both the label and a reasonable instruction, so I would probably do:

```
7. :guilabel:`Sign in with Aggregate password`

   username: ``aggregate``
   password: ``aggregate``

```

Reply:
username is now `administrator`

Reply:
Should I add more `:guilabel:` for other messages that the user will see on the console or on VirtualBox?

Reply:
Also, with that markup for the username and password, I get everything on one line... Is that intended? If so, maybe I'd have to apply the same markup for other username/password blocks on the same doc.